src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -319,13 +319,14 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <Target Name="_GetBuildOutputFilesWithTfm"
-          DependsOnTargets="BuiltProjectOutputGroup;DocumentationProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput)"
+          DependsOnTargets="BuiltProjectOutputGroup;DocumentationProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;$(TargetsForTfmSpecificBuildOutput);PrepareLibraryLayout"
           Returns="@(BuildOutputInPackage)">
     <ItemGroup>
       <BuildOutputInPackage Include="@(SatelliteDllsProjectOutputGroupOutput);
                             @(BuiltProjectOutputGroupOutput);
                             @(DocumentationProjectOutputGroupOutput);
-                            @(_PathToPriFile)"/>
+                            @(_PathToPriFile);
+                            @(_LayoutFile)"/>
```

Comment:
we shouldn't take dependencies on items that start with _ as they are meant to be internal.